### PR TITLE
Improvements to config.md

### PR DESCRIPTION
**Pull request by Jacalz**
This reorders the config option in alphabetical order as well as adding the option for adding sub-folders, as this is already in svtplay-dl itself. I also added my own configuration file as an example since it shows another useful use case of the config file in my opinion.

### CODE REVIEW

File: config.md [hunk 1]
```diff
@@ -94,3 +95,13 @@ service:
     filename: "{title}.s{season}e{episode}.{episodename}-{service}.{ext}"
 ```
 
+This will download all episodes with subtitles and audio merged in to the video, sort them in to subfolders within ~/svtplay-dl and finally write the file name without the service attached.
+```
```
Comment (Jacalz) [inline config.md]: - Need to add yaml markup language here

Comment (Jacalz) [reply config.md]: This is now fixed thanks to https://github.com/spaam/svtplay-dl-web/commit/fcc2a0be7a0b54f36987b145c9ad6dcc012aa6fe

**Comment by Jacalz** [comment]
cc @spaam for review

**Comment by spaam** [comment]
thank you :)

**Pull request merged**